lib/ipfs/IpfsError.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Standardized error class for throwing generic errors internal to this project.
+ */
+export default class IpfsError extends Error {
```

Comment:
Consider creating a bug to consolidating `IpfsError`, `BitcionError` and `SidetreeError` into one single class under `common`.

Reply:
https://github.com/decentralized-identity/sidetree/issues/417